recipes/seqstr/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: seqstr
+  version: 0.1.0
+
+source:
+  url: https://github.com/jzhoulab/Seqstr/archive/refs/tags/v0.1.0.tar.gz 
+  sha256: ddef0e2fce2f3c262bf23e16a45bf65886228bd20be314a05842f7f72bd9a705
+
+build:
+  number: 0
+  noarch: python
+  run_exports:
+    - {{ pin_subpackage('seqstr', max_pin="x.x") }} 
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip >=21.2.2
+  run:
+    - python >=3.7
+    - requests >=2.25.0
+    - pyfaidx >=0.6.3
+
+about:
+  home: https://github.com/jzhoulab/Seqstr
+  license: MIT
+  summary: Lightweight tool to compile simple string input into long genomic sequences
+  description: |
+    Seqstr is designed to provide a concise and flexible way to specify long genomic sequences that can be used for downstream analysis. For example, it can be used by web servers to avoid transferring long genomic sequences. Seqstr is also a format specification, which can be implemented in different languages. We also provide a test suite for verifying an implementation.
+
+extra:
+  recipe-maintainers:
+    - Chenlai Shi <shichenlai@gmail.com>
```

Comment:
I suggest using your GitHub username here instead so you can be pinged by the bot autobump PRs for future releases.